castai/resource_autoscaler.go

```diff
@@ -5,12 +5,12 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	jsonpatch "github.com/evanphx/json-patch"
 	"io"
 	"log"
 	"net/http"
 	"time"
 
-	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
nitpick: this moved to the wrong group for no reason

Reply:
Moved it back